### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Mindfulplays**
- Add test CLI doc
- Add github sync/push doc

### CODE REVIEW

**Comment by tomspilman** [comment]
@SimonDarksideJ @harry-cpp @mrhelmut  this seem ok to you all?

File: Tests/README.md [hunk 1]
```diff
@@ -9,15 +9,41 @@ properties, methods and behavior to guarantee compatibility with XNA in
...
+**Note** These are quick hints, for extensive help use `dotnet test --help`.
+
+```bash
+dotnet test ./Tests/MonoGame.Tests.DesktopGL.csproj -t
+```
+
+To run/filter specific classes/namespaces, an example:
+
+```bash
+dotnet test ./Tests/MonoGame.Tests.DesktopGL.csproj --filter MonoGame.Tests.Visual
```
Comment (harry-cpp) [inline Tests/README.md:45]: `./` is redundant, same in a couple places above.

File: README.md [hunk 2]
```diff
@@ -80,6 +80,19 @@ A high level breakdown of the components of the framework:
 * [mgfxc](Tools/MonoGame.Effect.Compiler) is the command line effect compiler tool.
 * The [mgcb-editor](Tools/MonoGame.Content.Builder.Editor) tool is a GUI frontend for content processing.
 
+### Syncing/Submitting PRs
+
+After the initial setup, use the following commands to sync to the latest changes.
+
+```bash
+git pull origin develop
```
Comment (harry-cpp) [inline README.md:88]: You actually wanna sync by doing `git pull --rebase origin develop` (I usually just call `git rebase -u origin develop` instead, might have made a mistake with the rebase version of pull command). This way your commits go on top of the upstream commits, and you don't create the "Merge branch develop of upstream" commits added.

This does however mean that pushing your changes back to your fork will require you to force push them.

**Review by harry-cpp — CHANGES_REQUESTED** [review]

File: Tests/README.md (hunk 1 above)
Comment (SimonDarksideJ) [reply Tests/README.md:45]: I missed these in update @Mindfulplays , if you can update.
@harry-cpp do you want to provide specifics for the comment regarding the merge path?

File: README.md (hunk 2 above)
Comment (Mindfulplays) [reply README.md:88]: Just seeing this now. Thanks for the `rebase upstream / origin` tip :)

**Review by Mindfulplays — COMMENTED** [review]
Resolved

File: README.md [hunk 3]
```diff
@@ -80,6 +80,21 @@ A high level breakdown of the components of the framework:
 * [mgfxc](Tools/MonoGame.Effect.Compiler) is the command line effect compiler tool.
 * The [mgcb-editor](Tools/MonoGame.Content.Builder.Editor) tool is a GUI frontend for content processing.
 
+### Syncing/Submitting PRs
```
Comment (harry-cpp) [inline README.md:83]: Thinking about it, we probably shouldn't have this in the README, this does not give detailed enough information for a new person and for someone who knows how to use git is not useful.

**Comment by SimonDarksideJ** [comment]
I don't know what has happened, but this PR seems broken now, it's not showing changes to the ReadMe's but to actual MonoGame class/interface files?

**Comment by Mindfulplays** [comment]
Will fix soon; amateur hour in working with GitHub.

**Comment by Mindfulplays** [comment]
I may have to switch over to an actual branch here and dump this PR. I made a newbie mistake of working on develop directly for this PR only as this was my first PR. (Perhaps explains the contents of the docs)

**Comment by Mindfulplays** [comment]
Moved to https://github.com/MonoGame/MonoGame/pull/8108

**Pull request closed without merging**